lib/widgets/write_comment.dart

```diff
@@ -83,15 +84,17 @@ class WriteComment extends HookWidget {
       ),
       body: ListView(
         children: [
-          ConstrainedBox(
-            constraints: BoxConstraints(
-                maxHeight: MediaQuery.of(context).size.height * .35),
-            child: SingleChildScrollView(
-              padding: const EdgeInsets.all(8),
-              child: preview,
+          if (preview != null) ...[
```

Comment:
analyzing `preview` initialization tells me it is either a `Column` widget or `body` where body is either `SizedBox` or `MarkdownText`, so `preview` is never null. Am I missing something?